[ERROR] AttributeError: 'Series' object has no attribute 'glims_is_validation'

```pytb
Traceback (most recent call last):  File "/var/task/lambda_code.py", line 1788, in lambda_handler    processing_df = submit_libraries_to_pieriandx(processing_df)  
File "/var/task/lambda_code.py", line 348, in submit_libraries_to_pieriandx    processing_df["submission_arn"] = processing_df.apply(  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/frame.py", line 9568, in apply    
return op.apply().__finalize__(self, method="apply")  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 764, in apply    return self.apply_standard()  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 891, in apply_standard    results, res_index = self.apply_series_generator()  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 907, in apply_series_generator    results[i] = self.f(v)  
File "/var/task/lambda_code.py", line 350, in <lambda>    if x.glims_is_validation is True  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/generic.py", line 5902, in __getattr__    return object.__getattribute__(self, name) | [ERROR] AttributeError: 'Series' object has no attribute 'glims_is_validation' Traceback (most recent call last):   
File "/var/task/lambda_code.py", line 1788, in lambda_handler     processing_df = submit_libraries_to_pieriandx(processing_df)  
File "/var/task/lambda_code.py", line 348, in submit_libraries_to_pieriandx     processing_df["submission_arn"] = processing_df.apply(  
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/frame.py", line 9568, in apply     return op.apply().__finalize__(self, method="apply")   
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 764, in apply     return self.apply_standard()   
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 891, in apply_standard     results, res_index = self.apply_series_generator()   
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/apply.py", line 907, in apply_series_generator     results[i] = self.f(v)   
File "/var/task/lambda_code.py", line 350, in <lambda>     if x.glims_is_validation is True   
File "/var/lang/lib/python3.11/site-packages/pandas-1.5.3-py3.11-linux-x86_64.egg/pandas/core/generic.py", line 5902, in __getattr__     return object.__getattribute__(self, name)

```

--- a/deploy/cttso-ica-to-pieriandx-cdk/Changelog.md
+++ b/deploy/cttso-ica-to-pieriandx-cdk/Changelog.md
@@ -3,6 +3,17 @@
 Changes in this log refer only to changes that make it to the 'main' branch and
 are nested under deploy/cttso-ica-to-pieriandx-cdk.  
 
+## 2023-08-12
+
+> Author: Alexis Lucattini
+> Email: [Alexis.Lucattini@umccr.org](mailto:alexis.lucattini@umccr.org)
+
+* Updated submission columns
+
+### Hot fix
+
+* Updated submission column (fixes https://github.com/umccr/cttso-ica-to-pieriandx/issues/123)
+
 ## 2023-08-09
 
 > Author: Alexis Lucattini

--- a/deploy/cttso-ica-to-pieriandx-cdk/lambdas/launch_available_payloads_and_update_cttso_lims_sheet/lambda_code.py
+++ b/deploy/cttso-ica-to-pieriandx-cdk/lambdas/launch_available_payloads_and_update_cttso_lims_sheet/lambda_code.py
@@ -184,6 +184,8 @@ def get_libraries_for_processing(merged_df) -> pd.DataFrame:
         "subject_id",
         "library_id",
         "portal_wfr_id",
+        "glims_is_validation",
+        "glims_is_research",
         "is_validation_sample",
         "is_research_sample",
         "redcap_is_complete"
@@ -332,6 +334,8 @@ def submit_libraries_to_pieriandx(processing_df: pd.DataFrame) -> pd.DataFrame:
       * library_id
       * portal_wfr_id
       * is_validation_sample
+      * glims_is_validation
+      * glims_is_research
       * is_research_sample
       * redcap_is_complete
     :return:

